### DISCUSSION TITLE
View control in the observer in the 3d-pannel

### DISCUSSION THREAD

**Discussion by AndyYangjd** [foxglove/community#283] [Q&A]
Just like the icons in CAD(solidworks, catia, ...), see picutre below:

![image](https://user-images.githubusercontent.com/44262434/176569809-c33a2463-901a-4c70-b305-e86b10d458fa.png)

![image](https://user-images.githubusercontent.com/44262434/176570002-4cd19266-e9c0-499d-8221-27c4438cffdc.png)

**Comment by jhurliman** [comment]
cc @2metres
